README.md

```diff
@@ -88,47 +88,56 @@ Supported versions:
 
-`aztfexport query [option] <arg where predicate>` exports a set of resources (and its including resources with `--recursive`) by an Azure Resource Graph [`where` predicate](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/whereoperator). Note that you can combine multiple conditions in one `where` predicate, e.g.  `resourceGroup =~ "my-rg" and type =~ "microsoft.network/virtualnetworks"`.
+### Query
 
-> 💡 Resource group mode is the same as running `aztfexport query --recursive "resourceGroup =~ 'my-rg'"`, except it also add on the resource group itself.
+`aztfexport query [option] <arg where predicate>` exports a set of resources (and its including resources with option flag `--recursive`) by an Azure Resource Graph [`where` predicate](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/whereoperator). Note that you can combine multiple conditions in one `where` predicate so long as you wrap the predicate in double quotes, e.g.  `aztfexport query "resourceGroup =~ 'my-rg' and type =~ 'microsoft.network/virtualnetworks'"`.
 
-`aztfexport` depends on `azlist`, which uses ARG behind the scenes. `azlist` will first make an ARG call with the given where predicate, then if `--recursive` is specified, it will recursively call the "LIST" on the known child resource types. Since ARG only returns ARM tracked resources at this moment, but not for the RP proxy resources (e.g. subnet, network security rules, storage containers, etc). If you uses predicate like `type =~ "microsoft.network/virtualnetworks/subnets"`, it returns you nothing since subnet is not an ARM tracked resource.
+> 💡 Resource group mode is the same as running `aztfexport query --recursive "resourceGroup =~ 'my-rg'"`, except it also exports the resource group itself.
 
-To workaround above, you can query with a bigger scope (e.g. `type =~ "microsoft.network/virtualnetworks"`) in interactive mode, then manually remove the resources other than subnets.
+`aztfexport` depends on `azlist`, which uses ARG behind the scenes. `azlist` will first make an ARG call with the given where predicate, then if `--recursive` is specified, it will recursively call "LIST" on the known child resource types. To read more about Azure Resource Graph visit the [ARG Documentation](https://learn.microsoft.com/en-us/azure/governance/resource-graph/).  
+As of now ARG only returns ARM tracked resources, but not RP proxy resources (e.g. subnet, network security rules, storage containers, etc). Thus, if you run query mode on proxy resources (e.g. `aztfexport query type =~ "microsoft.network/virtualnetworks/subnets"`), it will return nothing since subnet is not an ARM tracked resource. A workaround is to query with a bigger scope (e.g. `type =~ "microsoft.network/virtualnetworks"`) in interactive mode, then manually remove the resources other than subnets.
```

Comment:
```suggestion

As of now ARG only returns ARM tracked resources, but not RP proxy resources (e.g. subnet, network security rules, storage containers, etc). Thus, if you run query mode on proxy resources (e.g. `aztfexport query "type =~ 'microsoft.network/virtualnetworks/subnets'"`), it will return nothing since subnet is not an ARM tracked resource. A workaround is to query with a bigger scope (e.g. `type =~ 'microsoft.network/virtualnetworks'`) in interactive mode, then manually remove the resources other than subnets.
```